referenceframe/model.go

```diff
@@ -36,15 +36,14 @@ type SimpleModel struct {
 	// OrdTransforms is the list of transforms ordered from end effector to base
 	OrdTransforms []Frame
 	modelConfig   *ModelConfig
-	poseCache     *sync.Map
+	poseCache     sync.Map
```

Comment:
I understand `sync.Map`s have problems with copy assignment, hence the original pointer, but because we aren't using `poseCache` anywhere this currently works fine. Should we just remove it to prevent someone from causing new programming errors later? @biotinker thoughts?

Reply:
This works fine because the cache is accessed through pointer methods which is the common pattern. If this didn't work it would fail to go vet with the internal noCopy idiom

Reply:
But also if pose cache isn't being used I don't see why we have it in our code?

Reply:
OK, `go vet` letting this through makes sense. I believe it was removed because it ceased to give us a performance speed-up. Not sure why it wasn't fully removed at the time.

Reply:
Actually, this is still in `CachedTransform`. Thx GH diff search.